src/Features/Core/Portable/Navigation/NavigationOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Navigation
+{
+    internal static class NavigationOptions
+    {
+        public const string FeatureName = "Navigation";
+
+        public static readonly Option<bool> UsePreviewTab = new Option<bool>(FeatureName, "UsePreviewTab", defaultValue: false);
```

Comment:
So does this control "go to metadata as source instead of object browser" (i.e. the new option we're introducing) or more generally "never use the preview tab" which already reflects an existing VS flag?